crates/bevy_winit/src/winit_windows.rs

```diff
@@ -194,7 +194,7 @@ impl WinitWindows {
             use winit::platform::web::WindowExtWebSys;
 
             if window.canvas.is_none() {
-                let canvas = winit_window.canvas();
+                let canvas = winit_window.canvas().unwrap();
```

Comment:
Follow-up from https://github.com/bevyengine/bevy/pull/8745#discussion_r1387044986:
> Only returns the canvas if called from inside the window.

Apparently this didn't turn out very clear. This will only return `None` if called from a Web worker, it will always succeed if called from the "main thread" (the "window" context).